### DISCUSSION TITLE
Failed to set code signing settings

### DISCUSSION THREAD

**Discussion by PocketBomb** [codemagic-ci-cd/codemagic-docs#2893] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

1) I created update for app
2) After trying build I had this error
3) Tried recreate code signing and provision: FAILED
4) Tried to update xcode gem: the same

### Expected results

success build. I didn't have this error before.

### Actual results

<img width="974" alt="ewgewrgewr" src="https://github.com/user-attachments/assets/fec2dc76-fcb9-420e-b1cd-e9814037d1ed" />
Failed to set code signing settings for Pocket_Indicator.xcodeproj

### Build id (optional)

6764ae1a3b83fcb3b0aead4b

**Comment by dam-ease** [comment]
Hello @PocketBomb 
I noticed your recent build was successful, all good now right?
